src/rust/engine/build_utils/src/lib.rs

```diff
@@ -0,0 +1,143 @@
+use std::env;
```

Comment:
Throughout this file: I tend to try to `.unwrap()` only when things are guaranteed not to fail, and instead to `.expect("Message")` when things may fail, so that you have a little context. I think that could be useful everywhere that you're `.unwrap()`ing in this file.

Reply:
Obsoleted here, trimmed down API with the simplification of `bazel_protos/build.rs` now delegating to `cargo.sh` for `protoc` setup. That said, I followed this in `cffi_build.rs`.